bigframes/operations/semantics.py

```diff
@@ -1072,3 +1092,22 @@ def _validate_model(model):
 
         if not isinstance(model, GeminiTextGenerator):
             raise TypeError("Model is not GeminiText Generator")
+
+    @staticmethod
+    def _confirm_operation(row_count: int) -> bool:
```

Comment:
Let's have this raise instead of return. Otherwise, you'll have to change all the methods to return `Optional[Series]` / `Optional[DataFrame]` for type checking, which would be annoying to deal with for users in non-interactive use cases.